dsm/resource_aws_sobject.go

```diff
@@ -36,7 +35,18 @@ func resourceAWSSobject() *schema.Resource {
 		ReadContext:   resourceReadAWSSobject,
 		UpdateContext: resourceUpdateAWSSobject,
 		DeleteContext: resourceDeleteAWSSobject,
-		Description: "Creates a new security object in AWS KMS. This is a Bring-Your-Own-Key (BYOK) method and copies an existing DSM local security object to AWS KMS as a Customer Managed Key (CMK).The returned resource object contains the UUID of the security object for further references.",
+		Description: "Creates a new security object in AWS KMS. This is a Bring-Your-Own-Key (BYOK) method and copies an existing DSM local security object to AWS KMS as a Customer Managed Key (CMK).The returned resource object contains the UUID of the security object for further references.\n" +
+		"AWS sobject can also rotate and enable schedule deletion. For more examples, refer Guides/dsm_aws_sobject, Guides/rotate_with_AWS_option and rotate_with_DSM_option.\n\n" +
+		"**Temporary Credentials**: AWS security object can also be created using AWS temporary credentials. Please refer the below example for temporary credentials.\n\n" +
+		"**Note**: Once schedule deletion is enabled, AWS security object can't be modified.\n\n" +
+		"**Deletion of a dsm_aws_sobject**: Unlike dsm_sobject, deletion of a dsm_aws_sobject is not normal.\n\n" +
+		"**Steps to delete a dsm_azure_sobject:**\n" +
+		"   * Enable schedule_deletion as shown in the examples of guides/dsm_azure_sobject.\n" +
+		"   * Enable delete_key_material as shown in the examples of guides/dsm_azure_sobject.\n" +
+		"   * A dsm_aws_sobject can be deleted completely only when its state is `destroyed`.\n" +
+		"   * A dsm_aws_sobject comes to destroyed state when the key is deleted from Azure key vault.\n" +
```

Comment:
`...comes to destroyed state when the key ...` -> `... is destroyed when the key ...`

Reply:
Changed.